lib/lib_radarr.py

```diff
@@ -150,6 +152,10 @@ def setup_plex(self, hostname, port, user=None, passwd=None):
         host.service_start(self.service_name)
 
     def install_deps(self):
+        fetch.add_source("deb https://download.mono-project.com/repo/ubuntu stable-{series} main",
+                         key="3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF")
```

Comment:
We should move this to config.
If we really wanted to shave some yaks, we could create layer-mono... then move layer-sonarr to that, too.